sections/study_programs.md

```diff
@@ -0,0 +1,135 @@
+# CentrePoing Designer Study Programs
+
+**NOTE: These requests require the *CentrePoint* API scope. (see [Scopes](scopes.md))**
+
+## List of Study Programs
+
+Returns a list of study programs that are completed and not depreciated within your organization.
```

Comment:
deprecated

Reply:
Yes, I have corrected all the typos pertaining to deprecated.

The PR is now ready for your review!